database.rules.json

```diff
@@ -1,59 +1,83 @@
 {
```

Comment:
Brand new to Firebase, so was hard for me to code review... and it looks like some whitespace changes made the diff look larger.  It looks like some changes to support the Admin Interface are mixed in here also..?